### DISCUSSION TITLE
Request ORDERING field dropdown for Modules be same as Menu Links

### DISCUSSION THREAD

**Discussion by sdrawdenek** [joomla/joomla-cms#30047] [2. Cold Features]
### Steps to reproduce the issue

Open a Menu Link Item in Menu Manager and select ORDERING field - Image 1

Open a Module Item in Module Manager and select ORDERING field - Image 2

### Expected result

When open Module Manager and select ORDERING field I expected to see

- First -
- Last -

options as is the case in Menu Manager and select ORDERING field.

### Actual result

See image 2 - no First or Last option.

Could these please be added?

### System information (as much as possible)

J4 Dev 3![screen shot 2020-07-07 at 18 36 07](https://issues.joomla.org/uploads/1/5b45e9e1dc7907291c17eeaea790ee73.png)

### Additional comments

![screen shot 2020-07-07 at 18 36 07](https://issues.joomla.org/uploads/1/c23ca9c896f58beac289dea1c16d24c4.png)

**Comment by sdrawdenek** [comment]
Apologies if this on reflection is in the wrong category (issues).

If it should be in Pull Requests would someone oblige?

Don't think I can do this as a general contributor.

**Comment by ReLater** [comment]
"Issues" is correct. "Pull Requests" is for submitting code changes.

**Comment by ghost** [comment]
Issue need Label "J4-Issue".  Feature request make sense.
